addon/globalPlugins/EnhancedDictionaries/profileConfigurationHelper.py

```diff
@@ -0,0 +1,32 @@
+import config
+from logHandler import log
+
+module = 'EnhancedDictionaries'
+key = "keepUpdatedCheckbox"
+
+
+def stringToBool(value):
+	if value == "True":
+		return True
+	return False
+
+
+def getSavedCheckboxValueForProfile():
```

Comment:
Function name suggestion
```suggestion
def getSavedKeepDictionaryUpdatedCheckboxValueForProfile():
```